r2/r2/lib/eventcollector.py

```diff
@@ -85,6 +85,7 @@ def __init__(self, queue=amqp):
         self.queue = queue
 
     def save_event(self, event):
+        return # don't track events
```

Comment:
Should we make this configurable based on a boolean in the app globals? Just in case we decide we want to process those events in the future...

Reply:
That's going to be a lot more code touched and require more time to test...I'd suggest if we want to do that we'll know enough ahead of time to be able to remove this line.

Reply:
Makes sense to me. :+1: